index.html

```diff
@@ -1,117 +1,283 @@
+                    <div class="col-md-12">
+                        <h3 class="head-line">CHARITECH RULES</h3>
+                        
+                    </div>
+                </div>
+                <div class="row ">
+                      <ul class="media-list">
+                        <li class="media">
+            
+                        <div class="media-body">
+                          <p>
+                            The aim of ChariTech is to create something original for charity which can really make an impact. We've set out some rules to help the event go more smoothly for everyone.
+                          </p>
+                          <ul>
+                            <li>
+                              Please respect and look after the Computer Lab. Note that <span class="emphasis">no food or drinks are permitted in the Intel Laboratory</span>, and we will be checking!
+                            </li>
+                            <li>  
+                              You can work on your game in a team of up to four people.
+                            </li>
+                            <li>
+                              Hacking commences at 10.45 and finishes at 20:00.
+                            </li>                
+                            <li>
+                              You are free to use any tools or libraries available to you to create your product. You can start with any pre-existing code or content that you like and you are free to use third-party assets, as long as you let the judges know what you created yourself and what you didn't. <span class="emphasis">Failure to do so could risk disqualification.</span>
+                            </li>
+                            <li>
+                              It's your responsibilty to make sure that you have the right to use third-party assets (for example, that they are public domain or available under an appropriate license).
+                            </li>
+                            <li>
+                              Be kind and considerate to your fellow hackers and our volunteers. We're all here to have fun! By participating in the hackathon, you agree to abide by <a href="https://hackcodeofconduct.org/" target="_blank">this Code of Conduct</a>.
+                            </li>
+                          </ul>
+                        </div>  
+                        </li>                   
+ 
+                     </ul>                                            
+                </div>
+     </section>
+     <!-- RULES  SECTION END-->
+      <section id="contact-sec"  >
+        <div class="overlay">           
+          <div class="container basic-set">
+            <div class="row " >
+              <div class="col-md-4 col-sm-4" >
+                <h3>OUR LOCATION</h3>
+                <br />
+                <h4>William Gates Building, 15 J.J. Thomson Ave, Cambridge CB3 0FD</h4>
```

Comment:
Elsewhere I talk about semantic HTML. When writing `h*` tags, I find it useful to ask: "Is the text I'm about to write the heading of some section of text?". If the answer is no, I look for another tag. If using a heading tag provided me with some nice styles, I look for (or create) a class that I can use. I think bootstrap provides helpers to make things look like headers.